web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -390,6 +390,82 @@ $.extend(fixmystreet.set_up, {
     });
   },
 
+  manage_duplicates: function() {
+      // Deal with changes to report state by inspector/other staff, specifically
+      // displaying nearby reports if it's changed to 'duplicate'.
+      function refresh_duplicate_list() {
```

Comment:
OK, as discussed we'll have a bit of a tidy up and move inspector-specific JS into its own file that we only serve when it'll actually be used 👍